newton/_src/solvers/kamino/core/builder.py

```
@@ -0,0 +1,1335 @@
+        if uid is None:
+            uid = str(uuid.uuid4())
+
+        # Set the default material if not provided
+        if material is None:
+            material = self.materials.default.name
+
+        # Check if the body name and UID are unique
+        name_exists = name in world.collision_geom_names
+        uid_exists = uid in world.collision_geom_uids
+        if name_exists and uid_exists:
+            raise ValueError(f"Geometry name '{name}' and UID '{uid}' already exists.")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)